lib/itamae/logger.rb

```diff
@@ -48,10 +48,25 @@ def color(code, &block)
       %w!debug info warn error fatal unknown!.each do |level|
         module_eval(<<-EOC, __FILE__, __LINE__ + 1)
           def #{level}(msg)
-            super("  " * indent_depth + msg)
+            super(indent_msg(msg))
```

Comment:
`" " * indent_depth + super(msg)` returns just an indented string and it doesn't indent the message.
I have thought of introducing `Itamae::Logger::Message` to indent messages in `Formatter#msg2str`, but the logic of indentation should not depend on `Formatter`, because [this line](https://github.com/abicky/itamae/blob/feature/log-any-objects/lib/itamae/logger.rb#L136) accepts other loggers whose formatter is not `Itamae::Logger::Formatter`.